test/integration/reg_user_can_view_past_order_test.rb

```diff
@@ -0,0 +1,73 @@
+require 'test_helper'
+
+
+class RegUserCanViewPastOrderTest < ActionDispatch::IntegrationTest
+
+  # Background: An existing user that has one previous order
+
+  test 'a registered user can view past orders' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.